README.md

```diff
@@ -1,6 +1,11 @@
 # Qiskit IBM Provider
 
-[![License](https://img.shields.io/github/license/Qiskit/qiskit-ibm-provider.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)[![Push-Test](https://github.com/Qiskit/qiskit-ibm-provider/actions/workflows/main.yml/badge.svg)](https://github.com/Qiskit/qiskit-ibm-provider/actions/workflows/main.yml)[![](https://img.shields.io/github/release/Qiskit/qiskit-ibm-provider.svg?style=popout-square)](https://github.com/Qiskit/qiskit-ibm-provider/releases)[![](https://img.shields.io/pypi/dm/qiskit-ibm.svg?style=popout-square)](https://pypi.org/project/qiskit-ibm/)
+[![License](https://img.shields.io/github/license/Qiskit/qiskit-ibm-provider.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
I find this much easier to maintain if we use separate lines. It also appears to render correctly
https://github.com/Qiskit/qiskit-ibm-provider/blob/afebd4c160d1ea3d50db710f8fdfd8206caf3795/README.md